src/components/html-text-input.js

```diff
@@ -38,13 +38,30 @@ export class HTMLInputView extends React.Component<PropsType, StateType> {
 	textInput: TextInput;
 	edit: string => mixed;
 	stopEditing: () => mixed;
+	panResponder: PanResponder;
 
 	constructor() {
 		super( ...arguments );
 
 		this.edit = this.edit.bind( this );
 		this.stopEditing = this.stopEditing.bind( this );
 
+		this.panResponder = PanResponder.create( {
+			onStartShouldSetPanResponder: ( ) => true,
+			onStartShouldSetPanResponderCapture: ( ) => true,
```

Comment:
I think that just the `onStartShouldSetPanResponderCapture` is necessary for the pan responder to capture the event (and trigger `onPanResponderMove`).